src/main/java/com/webonise/rbs/controller/BookingController.java

```diff
@@ -21,49 +24,59 @@
     @Autowired
     private BookingService bookingService;
 
+    @Autowired
+    private RoomService roomService;
+
+    @Autowired
+    private EventService eventService;
+
     @GetMapping
-    public String showBookings(Model model) {
-        model.addAttribute("booking", new Booking());
-        model.addAttribute("allBookings", bookingService.getAllBookings());
+    public String showBookings (Model model) {
+        model.addAttribute("booking" , new Booking());
```

Comment:
Please follow the standard of code format, remove space before comma(,)
A comma (,) always stay attached to the token that precedes whether it's programming language or language of communication.